aviary/variable_info/variables.py

```diff
@@ -206,13 +207,16 @@ class Engine:
         MASS = 'aircraft:engine:mass'
         MASS_SCALER = 'aircraft:engine:mass_scaler'
         MASS_SPECIFIC = 'aircraft:engine:mass_specific'
-        MODEL_SLS_THRUST = 'aircraft:engine:model_sls_thrust'
```

Comment:
Does this need to be removed from variable_meta_data.py too?

Reply:
Just checked, weirdly no, it was never in the metadata